PlainTasks.py

```diff
@@ -501,13 +501,27 @@ class PlainTasksOpenLinkCommand(sublime_plugin.TextCommand):
         ''')
 
     def _format_res(self, res):
-        return [res[0], "line: %d column: %d" % (int(res[1]), int(res[2]))]
+        if res[3] == 'f':
+            return [res[0], "line: %d column: %d" % (int(res[1]), int(res[2]))]
+        elif res[3] == 'd':
+            return [res[0]]
 
     def _on_panel_selection(self, selection):
         if selection >= 0:
             res = self._current_res[selection]
             win = sublime.active_window()
-            self.opened_file = win.open_file('%s:%s:%s' % res, sublime.ENCODED_POSITION)
+            if (not ST2) and res[3] == 'd':
```

Comment:
No-no, if user is on ST2 and link points to directory, your code will try to open it as file.
It should be somewhat like this:

```python 
if ST2 and res[3] == 'd':
    return
elif res[3] == 'd':
    ...
```